backend/app/usecase/changelog/changelog_test.go

```diff
@@ -0,0 +1,208 @@
+// +build !integration all
+
+package changelog
+
+import (
+	"testing"
+	"time"
+
+	"github.com/short-d/app/mdtest"
+	"github.com/short-d/short/app/entity"
+	"github.com/short-d/short/app/usecase/keygen"
+	"github.com/short-d/short/app/usecase/repository"
+	"github.com/short-d/short/app/usecase/service"
+)
+
+func TestPersist_CreateChange(t *testing.T) {
+	t.Parallel()
+
+	summaryMarkdown := "summary"
```

Comment:
Let's put explicitly write out `summaryMarkdown` for each test case. Let's also add a case for `nil` summaryMarkdown.